### PR TITLE
Added clustergroup information in values files

### PR DESCRIPTION
**Pull request by hbisht-RH-ccs**
Added information related to ClusterGroup configuration in values files

### CODE REVIEW

**Comment by mbaldessari** [comment]
This is an automated message:

You can preview this docs PR at http://432.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

**Review by mhjacks — APPROVED** [review]
LGTM!

File: modules/basic-parameters-configuration-in-a-clustergroup.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* *Name*: Assigns a unique identifier to the ClusterGroup, acting as a reference point for identification within the broader infrastructure.
+
+* *TargetCluster*: Specifies the primary cluster(s) associated with the ClusterGroup, guiding management and control within its scope. The default is "in-cluster," which is a reasonable default. However, it can be changed when using the "push" model of OpenShift GitOps.
+
+* *IsHubCluster*: Indicates whether the designated cluster serves as a hub cluster, centralizing coordination and management within the ClusterGroup. Each Validated Pattern may have at most one cluster designated as a Hub cluster. This designation affects the use of Red Hat Advanced Cluster Management within the Validated Patterns framework.
+
+* *SharedValueFiles*: Enables specification of shared values files containing configuration settings shared across multiple clusters within the ClusterGroup, promoting consistency and simplifying configuration management. Each sharedValueFiles is applied as a value file to every application designated for installation in the clusterGroup.
+
```
Comment (kquinn1204) [inline modules/basic-parameters-configuration-in-a-clustergroup.adoc:16]: should sharedValueFiles be in backticks ?

File: modules/imperative-values-configuration-in-a-clustergroup.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+----
+
+In this example, the imperative section defines two jobs, "deploy-kubevirt-worker" and "configure-aap-controller".
+
+The "deploy-kubevirt-worker" job is responsible for ensuring that the cluster runs on AWS. It uses the OpenShift MachineSet API to add a baremetal node for running virtual machines.
+
+The "configure-aap-controller" job sets up the Ansible Automation Platform (AAP), a crucial component of the Ansible Edge GitOps platform. This job entitles AAP and sets up projects, jobs, and credentials. Unlike the default container image, this example uses a different image.
+
+Additionally, an optional `clusterRoleYaml` section is defined. By default, the imperative job runs under Role-based access control (RBAC), providing read-only access to all resources within its cluster. However, if a job requires write access to alter or generate settings, such permissions can be specified within the `clusterRoleYaml` section. In the AnsibleEdge scenario, the "deploy-kubevirt-worker" job needs permissions to manipulate (and create) machinesets, while the "configure-aap-controller" job requires read-only access to Kubernetes objects.
+
```
Comment (kquinn1204) [inline modules/imperative-values-configuration-in-a-clustergroup.adoc:58]: Do we need parentheses (), maybe  needs permissions to manipulate and create machinesets,

File: modules/managed-clustergroups-configuration-in-a-clustergroup.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
+:_content-type: CONCEPT
+:imagesdir: ../../images
+
+[id="managed-clustergroups-configuration-in-a-clustergroup"]
+= Managed cluster groups configuration in a ClusterGroup
+
+Configuring managed cluster groups within a ClusterGroup enhances the organizational structure and simplifies resource management. Through the "managedClusterGroups" parameter, you can define and organize clusters based on specific criteria, promoting efficient management and resource allocation within the ClusterGroup. This functionality streamlines management and coordination tasks across the infrastructure. Managed ClusterGroups mirror the configuration of a single cluster, facilitating the deployment of identical applications or implementing minor configuration adjustments across multiple clusters.
+
```
Comment (kquinn1204) [inline modules/managed-clustergroups-configuration-in-a-clustergroup.adoc:8]: Should we use `managedClusterGroups` in backticks

File: modules/namespace-configuration-in-a-clustergroup.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    config-demo:
+      labels:
+        app: 'config-demo'
+        tier: 'test'
+      annotations:
+        'test/annotation': true
+    hello-world: {}
+
+----
+In this example, we add “app” and “tier” labels to the config-demo app, and the annotation “test/annotation: true” to it. All other namespaces are created with defaults.
```
Comment (kquinn1204) [inline modules/namespace-configuration-in-a-clustergroup.adoc:50]: I am getting the impression now parameters are in "parameter_name"

File: modules/applications-configuration-in-a-clustergroup.adoc [hunk 5]
```diff
@@ -0,0 +1,63 @@
+:_content-type: CONCEPT
+:imagesdir: ../../images
+
+[id="applications-configuration-in-a-clustergroup"]
+= Applications configuration in a ClusterGroup
+
+Configuring applications within a ClusterGroup streamlines the process of defining, configuring, and coordinating software applications efficiently across clustered environments. The "applications" parameter allows you to specify the properties, dependencies, and resources required for each application deployment within the ClusterGroup, ensuring consistent and efficient deployment across multiple clusters.
+
```
[the remark this answers is not in the document]
Comment (hbisht-RH-ccs) [reply modules/applications-configuration-in-a-clustergroup.adoc:8]: Yes, I should use backticks. I will use them everywhere I have added the name of the parameter. Thanks!

File: modules/applications-configuration-in-a-clustergroup.adoc [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+In this context, applications refer to those defined by the OpenShift GitOps operator. These do not need to be applications in the traditional sense but are discrete units of deployment. The Validated Patterns framework supports any type of application that OpenShift GitOps can manage, but the most commonly used type is a Helm chart co-located in the repository that defines the pattern.
+
```
[the remark this answers is not in the document]
Comment (hbisht-RH-ccs) [reply modules/applications-configuration-in-a-clustergroup.adoc:10]: Done. Thanks!

File: modules/basic-parameters-configuration-in-a-clustergroup.adoc (hunk 1 above)
Comment (hbisht-RH-ccs) [reply modules/basic-parameters-configuration-in-a-clustergroup.adoc:16]: Yup, will do. Thanks!

File: modules/imperative-values-configuration-in-a-clustergroup.adoc (hunk 2 above)
Comment (hbisht-RH-ccs) [reply modules/imperative-values-configuration-in-a-clustergroup.adoc:58]: Opps... we don't need them. Appreciate your attention to detail! Thanks!

File: modules/managed-clustergroups-configuration-in-a-clustergroup.adoc (hunk 3 above)
Comment (hbisht-RH-ccs) [reply modules/managed-clustergroups-configuration-in-a-clustergroup.adoc:8]: Done. Thanks!

File: modules/namespace-configuration-in-a-clustergroup.adoc (hunk 4 above)
Comment (hbisht-RH-ccs) [reply modules/namespace-configuration-in-a-clustergroup.adoc:50]: I added backticks wherever I used parameters. Thanks!

**Pull request merged**